app/routes/news.mdx

```diff
@@ -22,6 +22,44 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2023
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'August 23, 2023'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>Important notes for GCN Kafka client users</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        Thanks to the feedback of some of our users, we have tracked down and fixed several reliability issues related to our Kafka brokers and clients. If you are consuming GCN Notices via Kafka in Python, then please take the actions noted below.
+
+        * **Daily Kafka outages have been resolved.** The GCN Kafka brokers had been experiencing daily outages of at around 00:00 UTC and lasting from about a minute to an hour. We traced the problem to a log collection agent that was using excessive memory. We resolved the issue by disabling the agent.
+        * **Python sample code has been updated.** Our [Python client sample code](/docs/client#python) was missing some necessary error handling. Some errors that may occur in the `consumer.consume()` loop are reported in the returned Kafka message. If you consume GCN Notices via Kafka in Python using the [gcn-kafka-python](https://github.com/nasa-gcn/gcn-kafka-python) client or [confluent-kafka-python](https://github.com/confluentinc/confluent-kafka-python), then please be sure to update your code as follows.
```

Comment:
some looks vague... 'some' necessary, 'Some' errors

Reply:
Well, that's because not _all_ errors in the `consume()` loop are reported that way. Some errors are reported by librdkafka by just printing them to the console.